perfmetrics/scripts/ml_tests/pytorch_dino_model/setup_host.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# Steps to bootstrap with gpu docker
+# Install docker
```

Comment:
nit: remove or change this comment to something like:
"Steps to install nvidia-driver, docker-engine, nvidia-docker2 and their required dependencies"